pepper-apis/dss-core/src/main/resources/org/broadinstitute/ddp/db/dao/JdbiBlockNesting.sql.stg

```diff
@@ -1,30 +1,72 @@
-  join block_type as bt on bt.block_type_id = nested.block_type_id
-  join revision as rev on rev.revision_id = bn.revision_id
-  left join block__expression as bve on bve.block_id = nested.block_id
-  left join expression as ev on ev.expression_id = bve.expression_id
-  left join revision as bve_rev on bve_rev.revision_id = bve.revision_id
-  left join block_enabled_expression as bee on bee.block_id = nested.block_id
-  left join expression as ee on ee.expression_id = bee.expression_id
-  left join revision as bee_rev on bee_rev.revision_id = bee.revision_id
-  join activity_instance as ai on ai.activity_instance_guid = :instanceGuid
- where bn.parent_block_id = :blockId
-   and rev.start_date \<= ai.created_at
-   and (rev.end_date is null or ai.created_at \< rev.end_date)
-   and (bve.block__expression_id is null or
-       (bve_rev.start_date \<= ai.created_at and (bve_rev.end_date is null or ai.created_at \< bve_rev.end_date)))
-   and (bee.block_enabled_expression_id is null or
-       (bee_rev.start_date \<= ai.created_at and (bee_rev.end_date is null or ai.created_at \< bee_rev.end_date)))
- order by bn.display_order asc
+SELECT
+    null AS section_id,
+    bn.parent_block_id AS parent_block_id,
+    bt.block_type_code AS block_meta_block_type_code,
+    nested.block_id AS block_meta_block_id,
+    nested.block_guid AS block_meta_block_guid,
+    ev.expression_id AS sh_expr_expression_id,
+    ev.expression_guid AS sh_expr_expression_guid,
+    ev.expression_text AS sh_expr_expression_text,
+    ee.expression_id AS en_expr_expression_id,
+    ee.expression_guid AS en_expr_expression_guid,
+    ee.expression_text AS en_expr_expression_text
+FROM block_nesting AS bn
+    JOIN block AS nested ON nested.block_id = bn.nested_block_id
+    JOIN block_type AS bt ON bt.block_type_id = nested.block_type_id
+    JOIN revision AS rev ON rev.revision_id = bn.revision_id
+    LEFT JOIN block__expression AS bev ON bev.block_id = nested.block_id
+    LEFT JOIN expression AS ev ON ev.expression_id = bev.expression_id
+    LEFT JOIN revision AS bev_rev ON bev_rev.revision_id = bev.revision_id
+    LEFT JOIN block_enabled_expression AS bee ON bee.block_id = nested.block_id
+    LEFT JOIN expression AS ee ON ee.expression_id = bee.expression_id
+    LEFT JOIN revision AS bee_rev ON bee_rev.revision_id = bee.revision_id
+    JOIN activity_instance AS ai ON ai.activity_instance_guid = :instanceGuid
+WHERE bn.parent_block_id = :blockId
+    AND rev.start_date \<= ai.created_at
+    AND (rev.end_date IS null OR ai.created_at \< rev.end_date)
+    AND (bev.block__expression_id IS null
+        OR (bev_rev.start_date \<= ai.created_at
+            AND (bev_rev.end_date IS null OR ai.created_at \< bev_rev.end_date)))
+    AND (bee.block_enabled_expression_id IS null
+        OR (bee_rev.start_date \<= ai.created_at
+            AND (bee_rev.end_date IS null OR ai.created_at \< bee_rev.end_date)))
+ORDER BY bn.display_order asc
```

Comment:
The only substantiative changes to this statement is in the `SELECT` part to support the nested `shownExpression`, `enabledExpression` and `blockDto` objects in the `FormBlockDto` class. The rest is just formatting changes.